tools/webpack/blocks.js

```diff
@@ -133,6 +133,7 @@ module.exports = function (
 		externalsType: 'module',
 		externals: {
 			'@wordpress/interactivity': '@wordpress/interactivity',
+			'@wordpress/interactivity-router': 'import @wordpress/interactivity-router',
 		},
 		module: {
 			rules: [
```

Comment:
@sirreal, can we remove Babel loader now that we use view files from `build-module`?

Reply:
By the way, the list of view files could be collected using `glob` so we don't have to maintain the list.

Reply:
Great feedback, I removed babel, I didn't use "glob" yet instead preferred an array of interactive blocks (just in case view is used for something else or something). I feel like that's a good start.

Reply:
This block looks like the only thing babel was doing was applying preset-react which does seem redundant at this point. 👍 to remove.

https://github.com/WordPress/wordpress-develop/blob/cf8b74de164f322a78ea51921b236213805e6f79/tools/webpack/blocks.js#L143-L158